admin/sections/config/providers/ollama.tsx

```diff
@@ -38,6 +38,7 @@ export const OllamaForm = ({
   defaultValues: any;
 }) => {
   const [models, setModels] = useState<string[]>();
+  const [loading, setLoading] = useState(true);
```

Comment:
@marcusschiesser
Just a minor thing. I added a loading state here to avoid showing error message while the api is fetching. But it is not the best solution because we're calling the fetch api for each time this form is rendered.
We can hold the provider model in parent data, but it requires us to change the current form data type. How about using the useQuery hook for this (set a staleTime to avoid calling the fetchModels)?

Reply:
yes why not?